### PR TITLE
Added UnionML Swags

### PR DESCRIPTION
**Pull request by sshiv5768**

### CODE REVIEW

**Comment by sshiv5768** [comment]
Hey @monizb check this out

**Comment by monizb** [comment]
@sshiv5768 any idea why the build is failing?

**Comment by sshiv5768** [comment]
@monizb no idea. I am just adding new item in list and making a PR. Nothing other than that

**Comment by sshiv5768** [comment]
I just checked  failed  build ``logs`` and i found this :
``SyntaxError: Unexpected string in JSON at position 6412 while parsing '{
  "status": true,
4:24:07 PM:   "list": {
4:24:07 PM:     "A":'
    at JSON.parse (<anonymous>)
4:24:07 PM: error Command failed with exit code 1. (https://ntl.fyi/exit-code-1)
4:24:07 PM: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
4:24:07 PM: 
``

**Comment by sshiv5768** [comment]
@monizb  do you found any solution?

**Comment by su-shubham** [comment]
If you fix the bug, commit your changes using **git commit -m "FIxed"** after that  **git push -f origin HEAD**

**Comment by sshiv5768** [comment]
@monizb I solved the issue it was just "," missing in the swags array 😁

File: src/list.json [hunk 1]
```diff
@@ -151,7 +151,16 @@
         "link": "https://blog.tooljet.com/hacktoberfest-2022/"
       }
     ],
-    "U": [],
+    "U": [
```
Comment (thefool76) [inline src/list.json:154]: hey @sshiv5768 you need to Re-Style it via running this command in the repo directory
`git checkout <your branch> `
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2336040/patch | git am`
`git push`

**Review by thefool76 — CHANGES_REQUESTED** [review]

File: src/list.json (hunk 1 above)
Comment (sshiv5768) [reply src/list.json:154]: Ok Will do it.

Comment (sshiv5768) [reply src/list.json:154]: @thefool76 check new commits

**Comment by sshiv5768** [comment]
@173 created a new PR with branch changes. @thefool76

**Pull request merged**